File: Java/2023-2/Estructura de Datos/ListasEnlazadasFinal/src/controller/AplMain.java
package controller;

import model.MyLinkedList;

public class AplMain {

	public static void main(String[] args) {
		MyLinkedList<Long> fico = new MyLinkedList<>();
		fico.add(18l); // [18]
		fico.add(13l); // [13, 18]
		fico.add(11l); // [11, 13, 18]
		fico.addLast(7l); // [11, 13, 18, 7]
		fico.addLast(69l);// [11, 13, 18, 7, 69]
		fico.addLast(420l); // [11, 13, 18, 7, 69, 420]
		fico.addLast(8l); // [11, 13, 18, 7, 69, 420, 8]
		System.out.println(fico.toString());
		
		
	}

}

File: CPP/ListasEnlazadasCPP/AplMain.cpp
#include <iostream>
#include "Node.cpp"
#include "MyLinkedList.cpp"
#include "GummyBrandDAO.cpp"
#include "GummyBrandDTO.cpp"
using namespace std;

int main(){

    MyLinkedList<int> fico;
    fico.add(18);
    fico.add(13);
    fico.add(11);
    fico.addLast(7);
    fico.addLast(69);
    fico.addLast(420);
    fico.addLast(8);

    cout << fico.toString() << endl;

    GummyBrandDAO listaDeMarcas;

    listaDeMarcas.agregar(GummyBrandDTO("Trululu", "Cocodrilo", 2.000));
    cout << listaDeMarcas.mostrar() << endl;
    
    return 0;
}